PayPal/src/main/java/com/braintreepayments/api/PayPalCheckoutRequest.java

```diff
@@ -177,7 +177,9 @@ String createRequestBody(Configuration configuration, Authorization authorizatio
 
         String billingAgreementDescription = getBillingAgreementDescription();
         if (shouldRequestBillingAgreement && !TextUtils.isEmpty(billingAgreementDescription)) {
-            parameters.put(DESCRIPTION_KEY, billingAgreementDescription);
```

Comment:
random q: was this the wrong key? I wish I could :actual-lol: here lol

Reply:
The key was write - we were just missing a layer of nesting in the JSON